src/components/Todo/Todo.tsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from 'react';
+import { TodoProps } from '../../types/ComponentsProps';
+
+export const Todo: React.FC<TodoProps> = ({ todo, deleteTodo }) => {
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    if (todo.id === 0) {
+      setIsLoading(true);
+    }
+  }, [todo.id]);
+
+  const handleDelete = () => {
+    setIsLoading(true);
+    deleteTodo(todo.id);
+  };
+
+  return (
+    <>
+      <div
+        data-cy="Todo"
+        className={`todo ${todo.completed ? 'completed' : ''}`}
+        key={todo.id}
+      >
+        <label className="todo__status-label" htmlFor={`todo-${todo.id}`}>
+          <input
+            id={`todo-${todo.id}`}
+            data-cy="TodoStatus"
+            type="checkbox"
+            className="todo__status"
+            checked={todo.completed}
```

Comment:
To prevent the above lint error pointed by GitHub, instead of using _id_ you should use the _name_ property for this input
(htmlFor on the label tracks the name property on an input)